app/src/candid/candid_parser.c

```diff
@@ -519,10 +561,52 @@ parser_error_t readCandidManageNeuron(parser_tx_t *tx, const uint8_t *input, uin
 
         switch (val->command.hash) {
             case hash_command_Split: {
+                // Check sanity Split
```

Comment:
Not now, but we should eventually refactor and move this part of the code to another file that deals with Commands.
As we deal with more and more types, we should start grouping and refactoring.
Let's open a ticket about this.